tools/extra/fpgabist/bist_common.py

```diff
@@ -24,6 +24,7 @@
 # ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,  EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 
+from distutils.spawn import find_executable
```

Comment:
You still need to update your PR, so you can remove this import. You're not using it anymore.